templates/component.ejs

```diff
@@ -13,15 +13,16 @@ const styles = StyleSheet.create({
 });
 
 type Props = {};
+<% var typeOfComponent = props.isPure ? 'PureComponent' : 'Component' %>
+<% if (props.isConnected) { %>class <%= props.name %> extends <%= typeOfComponent %><Props> {<% } else { %>export default class <%= props.name %> extends <%= typeOfComponent %><Props> {<% } %>
```

Comment:
Same thing could be applied for `export default class` vs `class` if `props.isConnected`